guides/hosting/installation-updates/deployments/databaseless.md

```diff
@@ -0,0 +1,71 @@
+# Building assets of administration and storefront without a database
+
+It is common to prebuilt assets in professional deployments to deploy the finished assets to the production environment. This task is mostly done by an CI job which doesn't have access to the the production database. Shopware needs access to the database to look up the installed extensions / loading the configured theme variables. To be able to build the assets without a database, we can use static dumped files.
+ This guide requires Shopware 6.4.0 or higher.
```

Comment:
Could you please mention the version requirement at the top of the article (right below the first heading) as described in this article: https://developer.shopware.com/docs/resources/guidelines/documentation/writing#maintaining-versions